source/main.cpp

```diff
@@ -164,8 +169,8 @@ int main(int argc, char *argv[])
 			// then check the default state of the universe.
 			if(!player.LoadRecent())
 				GameData::CheckReferences();
-			cout << "Parse completed." << endl;
-			return 0;
+			cout << "Parse completed with " << (hasErrors ? "more than one" : "no") << " error(s)." << endl;
```

Comment:
What if there was exactly one error?
```suggestion
			cout << "Parse completed with " << (hasErrors ? "at least one" : "no") << " error(s)." << endl;
```